componentTests/MigrationTest.cpp

```diff
@@ -248,7 +247,7 @@ SUITE(MigrationTest)
 
```

Comment:
On line 246, you new PSEUDO_DES and never delete it.  I suggest creating a local variable instead of new'ing it.

Reply:
I'm deleting it. I couldn't figure out how to make RandomFake work for this.